model/testing/src/icon4py/model/testing/datatest_utils.py

```diff
@@ -11,66 +11,24 @@
+from icon4py.model.testing import definitions, serialbox
 
 
-if TYPE_CHECKING:
-    from icon4py.model.testing import serialbox
+GLOBAL_EXPERIMENT = definitions.Experiments.EXCLAIM_APE.name
+REGIONAL_EXPERIMENT = definitions.Grids.MCH_CH_R04B09_DSL.name
+# we were using `REGIONAL_EXPERIMENT` for both grid and experiment, therefore we make sure their names match
+assert definitions.Grids.MCH_CH_R04B09_DSL.name == definitions.Experiments.MCH_CH_R04B09.name
+JABW_EXPERIMENT = definitions.Experiments.JW.name
+GAUSS3D_EXPERIMENT = definitions.Experiments.GAUSS3D.name
+WEISMAN_KLEMP_EXPERIMENT = definitions.Experiments.WEISMAN_KLEMP_TORUS.name
 
-GLOBAL_EXPERIMENT = "exclaim_ape_R02B04"
-REGIONAL_EXPERIMENT = "mch_ch_r04b09_dsl"
-R02B04_GLOBAL = "r02b04_global"
-R02B07_GLOBAL = "r02b07_global"
-ICON_CH2_SMALL = "mch_opr_r4b7"
-REGIONAL_BENCHMARK = "opr_r19b08"
-JABW_EXPERIMENT = "jabw_R02B04"
-GAUSS3D_EXPERIMENT = "gauss3d_torus"
-WEISMAN_KLEMP_EXPERIMENT = "weisman_klemp_torus"
-
-TORUS_100X116_1000M_GRID_URI = "https://polybox.ethz.ch/index.php/s/yqvotFss9i1OKzs/download"
-TORUS_50000x5000_RES500_GRID_URI = "https://polybox.ethz.ch/index.php/s/eclzK00TM9nnLtE/download"
-
-
-# GRID URIs for global grids
-#: small global grid use in exclaim_ape, JABW test case: num_cells = 20480
-#  the origin of this file is unclear (source = icon-dev)
-R02B04_GLOBAL_GRID_URI = "https://polybox.ethz.ch/index.php/s/BRiF7XrCCpGqpEF/download"
-
-#: large grid that should fit a single GPU for a simple atmospheric test case (Jablonowski-Williamson): num_cells = 1310720
-#: this is generated by the grid_generator of MPI-M
-R02B07_GLOBAL_GRID_URI = "https://polybox.ethz.ch/index.php/s/RMqNbaeHLD5tDd6/download"
-
-# GRID URIS for regional grids (MCH)
-# those files are generated by (different versions of) icontools, (DWD grid file generator), see source attribute in the file.
-
-#: used in mch_icon-ch2 experiment (2km resolution MCH production runs): num_cells = 283876
-
-R19_B07_MCH_LOCAL_GRID_URI = "https://polybox.ethz.ch/index.php/s/tFQian4aDzTES6c/download"
-
-#: used in mch_icon-ch2-small experiment (for valdiation): num_cells = 10700
-MCH_OPR_R04B07_DOMAIN01_GRID_URI = "https://polybox.ethz.ch/index.php/s/ZL7LeEDijGCSJGz/download"
-
-#: used in opr_r19b08 ("old" experiment) that should fit and saturate a single GPU run: num_cells = 44528
-DOMAIN01_GRID_URI = "https://polybox.ethz.ch/index.php/s/P6XfWcYjnrsNmeX/download"
-
-#: used in legacy mch_ch_rO4b09_dsl experiment: (num_cells = 20896)
-MC_CH_R04B09_DSL_GRID_URI = "https://polybox.ethz.ch/index.php/s/hD232znfEPBh4Oh/download"
-
-
-GRID_URIS = {
-    REGIONAL_EXPERIMENT: MC_CH_R04B09_DSL_GRID_URI,
-    R02B04_GLOBAL: R02B04_GLOBAL_GRID_URI,
-    R02B07_GLOBAL: R02B07_GLOBAL_GRID_URI,
-    ICON_CH2_SMALL: MCH_OPR_R04B07_DOMAIN01_GRID_URI,
-    REGIONAL_BENCHMARK: DOMAIN01_GRID_URI,
-    WEISMAN_KLEMP_EXPERIMENT: TORUS_50000x5000_RES500_GRID_URI,
-}
 
+# TODO(havogt): is this still needed?
```

Comment:
probably not. We changed the type of the grid.id to `str`, so where it is needed for the grid construction other than the `GridManager` that reads it from the file attributes, you could also use the `Grid.name`, if we want to be more accurate on what it is (and be able to really validate against Grids constructed from the file then we should add it to `definitions.testing.Grid`

Reply:
I will think about it in a later PR, should I keep the TODO or remove it?

Reply:
I would only remove the `TODO` together with the `GRID_IDS`...